### DISCUSSION TITLE
Cross analyse multiple pdfs

### DISCUSSION THREAD

**Discussion by CK92149** [mayooear/ai-pdf-chatbot-langchain#22] [General]
Is it feasible to add multiple pdf documents to the docs folder to cross analyse documents?

I have around 30 pdf documents worth of documentation. I'd like to be able to ask a single question where the correct answer spans multiple documents.

**Comment by halfprice06** [comment]
Could you combine them all into a single pdf?

**Reply by CK92149** [reply]
> Could you combine them all into a single pdf?

I'll give it a try. The reason I ask is because a (single) document can be updated or added and overwriting it would make life easier. 

is there a limit to a single file size?

Thanks!

edit, testing now with 1500 (combined) pages. Takes a while to ingest.

**Comment by dalkommatt** [comment]
It works with multiple PDFs already if you change the filePath in ingest-data.ts and ingest them one at a time. I would like to see it allow for subfolders inside of '/docs' and then allow the user to switch subfolder or _topics_. It would be nice if it automatically handled multiple PDFs too.

**Reply by CK92149** [reply]
> It works with multiple PDFs already if you change the filePath in ingest-data.ts and ingest them one at a time. I would like to see it allow for subfolders inside of '/docs' and then allow the user to switch subfolder or _topics_. It would be nice if it automatically handled multiple PDFs too.

Thanks, that seems like the most sensible way to do it at the moment. Change the (pdf)filepath after each ingestion.

**Comment by mayooear** [comment]
> Is it feasible to add multiple pdf documents to the docs folder to cross analyse documents?
> 
> I have around 30 pdf documents worth of documentation. I'd like to be able to ask a single question where the correct answer spans multiple documents.

I'm releasing a video on this tomorrow or tuesday. Feature will be added later in the week
